pom.xml

```diff
@@ -201,20 +201,20 @@
                                 <!-- Services used in test suite -->
                                 <keycloak.image>quay.io/keycloak/keycloak:16.1.0</keycloak.image>
                                 <rhsso.73.image>registry.access.redhat.com/redhat-sso-7/sso73-openshift</rhsso.73.image>
-                                <postgresql.13.image>quay.io/bitnami/postgresql:13.5.0</postgresql.13.image>
-                                <elastic.71.image>quay.io/bitnami/elasticsearch:7.10.2</elastic.71.image>
-                                <mysql.57.image>quay.io/bitnami/mysql:5.7.32</mysql.57.image>
-                                <mysql.bitnami.80.image>quay.io/bitnami/mysql:8.0</mysql.bitnami.80.image>
+                                <postgresql.13.image>postgres:13.6</postgresql.13.image>
+                                <elastic.71.image>elasticsearch:7.17.2</elastic.71.image>
+                                <mysql.57.image>mysql:5.7.36</mysql.57.image>
```

Comment:
Let's go with `5.7` tag, I don't fancy locking to micros - this makes us need to upgrade whenever there's a new tag (including security fixes)

Reply:
There is a problem with 5.7 tag: https://github.com/docker-library/mysql/issues/844

Reply:
Kindly add a note and TODO about it,

Reply:
Kindly add a TODO with reference.